src/__tests__/core/web_api/p2_api.test.js

```diff
@@ -86,7 +102,6 @@ describe('Auth0APIClient', () => {
               name: 'ExamplePlugin'
             }
           ],
-          _telemetryInfo: { newOption: true },
```

Comment:
shouldn't you keep this to show that it's being ignored?

Reply:
I removed from here because I tested all the scenarios in the other test, so there's no point in validating the telemetry here again